iceoryx_hoofs/test/CMakeLists.txt

```diff
@@ -58,7 +59,18 @@ target_include_directories(${PROJECT_PREFIX}_moduletests PRIVATE .)
 # TODO: fix conversion warnings
 target_compile_options(${PROJECT_PREFIX}_moduletests PRIVATE ${TEST_CXX_FLAGS})
 target_link_libraries(${PROJECT_PREFIX}_moduletests ${TEST_LINK_LIBS})
-set_target_properties(${PROJECT_PREFIX}_moduletests PROPERTIES 
+set_target_properties(${PROJECT_PREFIX}_moduletests PROPERTIES
+    CXX_STANDARD_REQUIRED ON
+    CXX_STANDARD ${ICEORYX_CXX_STANDARD}
+    POSITION_INDEPENDENT_CODE ON
+)
+
+# mock tests
+add_executable(${PROJECT_PREFIX}_mocktests ${MOCKTESTS_SRC})
+target_compile_options(${PROJECT_PREFIX}_mocktests PRIVATE ${TEST_CXX_FLAGS})
+target_include_directories(${PROJECT_PREFIX}_mocktests PRIVATE .)
+target_link_libraries(${PROJECT_PREFIX}_mocktests ${TEST_LINK_LIBS})
+set_target_properties(${PROJECT_PREFIX}_mocktests PROPERTIES
```

Comment:
This does not make sense if it does not run on the CI. Please either enable this for the CI or just add it to the moduletests